intelij-plugin/build.gradle

```diff
@@ -1,16 +1,38 @@
 plugins {
-    id 'org.jetbrains.intellij' version '1.3.0'
+    id 'org.jetbrains.intellij' version "$intellijGradlePluginVersion"
+    id 'org.jetbrains.changelog' version "$changelogVersion"
 }
 
+apply plugin: 'org.jetbrains.changelog'
+
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
-    version = '2021.2.3'
+    version = ideaVersion
     plugins = ['com.intellij.java']
 }
 
-patchPluginXml {
-    sinceBuild = '211'
-    untilBuild = ''
+changelog {
+    path = "${project.projectDir}/CHANGELOG.md"
+    header = "${-> version.get()}"
+    headerParserRegex = ~/(\d+\.\d+)/
+    itemPrefix = "*"
+    keepUnreleasedSection = true
+    unreleasedTerm = "[Unreleased]"
+    groups = []
+}
+
+tasks {
+
+    patchPluginXml {
+        sinceBuild = '211'
+        untilBuild = ''
+        changeNotes = changelog.getLatest().toHTML() + "<a href=\"https://github.com/PANTHEONtech/YANGinator/blob/master/intelij-plugin/CHANGELOG.md\">more...</a>"
+    }
+}
+
+runPluginVerifier {
+    ideVersions = ["IIC-2021.1.3", "IIU-2021.1.3", "PCC-2021.1.3", "PCP-2021.1.3", "PCE-2021.1.3", "DG-2021.1.3", "PS-2021.1.3", "RD-2021.1.3",
```

Comment:
cannot be this formatted in a way that every version is on new line? also it could be at the bottom of file

Reply:
Ok. And what do u think about updating that array to something like this:
- add variable version (for this case version = '2021.1.3'
- and then elements (all IDEs) in array will be dependent on version

Reply:
good idea but are you sure that all jetbrain platforms are released at same time with same versions?
if yes, then go for it

Reply:
an idea: could we move version definition into separate file, for example "compatible.version" each version separated by new line, all versions explicitly defined (so no "add variable version (for this case version = '2021.1.3'") which will make it crystal clear to read&understand and as a cherry on top, add there no only oldest version that we want to support but also EAP for every jetbrains product so we will ensure compatibility with new verisions as well